src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added support for the .NET7 version of the
```

Comment:
might be better to call this just 7.0 of DS.

Reply:
merging now. Will need to update the changelog to indicate what this hotfix is for.